spec/quickbooks-payments/basic_model_spec.rb

```diff
@@ -10,7 +15,7 @@ def _fakekey=(val)
 RSpec.describe Quickbooks::Payments::BasicModel do
   describe 'class methods' do
     describe 'initialize' do
-      let(:options) { { "_fakekey" => :val } }
+      let(:options) { { '_fakekey' => :val } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.